request.cpp

```diff
@@ -9,7 +10,7 @@ namespace server {
 //least changes. This should probably be changed in the future.
 std::unique_ptr<Request> Request::Parse(const std::string& raw_request)
 {
-	std::array<char, 16384ul> raw_request_;
+	std::array<char, 30000ul> raw_request_;
```

Comment:
Thanks for the catch! We will definitely look into this and make the according changes.